src/parsley/__init__.py

```
@@ -12,3 +21,28 @@
+    "parse_fields",
+    "parse",
+    "parse_board_type_id",
+    "parse_board_inst_id",
+    "parse_bitstring",
+    "parse_live_telemetry",
+    "parse_usb_debug",
+    "parse_logger",
+    "format_line",
+    "encode_data",
+    "calculate_msg_bit_len",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)